src/dashboard/Grafana/README.md

```diff
@@ -0,0 +1,68 @@
+## deploy grafana with helm
```

Comment:
we should remove this altogether along with sock-shop-live-dashboard.json

Reply:
removed the Grafana folder.